docker-app/worker_wrapper/wrapper.py

```diff
@@ -408,8 +408,7 @@ def after_docker_run(self) -> None:
         )
 
         try:
-            project_id = str(self.job.project.id)
-            package_ids = storage.get_stored_package_ids(project_id)
+            package_ids = storage.get_stored_package_ids(self.job.project)
             job_ids = [
```

Comment:
we can use sets and and values_lists with flat to make it more efficient

Reply:
It is not clear to me what you mean by "make it more efficient" by using sets?

I would suggest that we don't change too much on these functions as they were serving the codebase well so far.

What  I see as potential improvement here is indeed using `value_list("id", flat=True)`, but this is out of scope of this PR. Personally I would prefer to keep it as it is. But I will also accept a follow-up PR that changes things in the proposed way.